frontend/src/App.tsx

```diff
@@ -1,13 +1,42 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
-import {Api} from './components/Api.component';
+import  {ToastContainer} from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css'
+import { Api } from './components/Api.component';
+import { Header } from './components/Header.component';
 
 function App() {
-    return (
-        <div className="App">
-                <Api />
-        </div>
-    );
+  const [theme, setTheme] = useState<'light' | 'dark'>(() => {
```

Comment:
Can you move themes into constant file?

```suggestion
  enum Themes = {DARK='dark'=LIGHT: 'light'}
  const [theme, setTheme] = useState<Themes>(() => {
```

Also, replace all occurencies of hardcoded themes in code